CHANGES.md

```diff
@@ -3,6 +3,10 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.308
```

Comment:
You need to update the version in `version.go`

Reply:
Missed that. Fixed